--- dar-2.1.0/configure.ac.orig	2004-03-02 21:46:54.000000000 +0100
+++ dar-2.1.0/configure.ac	2004-03-02 21:50:21.000000000 +0100
@@ -151,8 +151,8 @@
 AC_ARG_ENABLE( [debug],
                 AC_HELP_STRING(--enable-debug, [build targets with debugging option and no optimization]),
                 [
-	                CXXFLAGS="-g -Wall"
-                        CFLAGS="-g -Wall"
+			CXXFLAGS="$CXXFLAGS -Wall"
+			CFLAGS="$CFLAGS -Wall"
                         debug_static="yes"
                 ],
                 [
