--- dar-2.5.9/configure.ac.orig	2017-01-21 11:59:32.000000000 +0100
+++ dar-2.5.9/configure.ac	2017-03-30 18:00:31.489079633 +0200
@@ -30,7 +30,6 @@
 AC_PROG_CC
 AC_PROG_LIBTOOL
 AC_PROG_MAKE_SET
-AC_PROG_RANLIB
 
 AC_MSG_CHECKING([for C++ compiler usability])
 AC_COMPILE_IFELSE([AC_LANG_PROGRAM([],
@@ -888,7 +887,7 @@
                 AC_HELP_STRING(--enable-debug, [build targets with debugging option and no optimization]),
                 [
 		# yes we modify user variable here to remove any optimization
-                        CXXFLAGS="$CXXSTDFLAGS -ggdb -Wall"
+                        CXXFLAGS="$CXXFLAGS -ggdb -Wall"
                         CFLAGS="-ggdb -Wall"
 			LDFLAGS="-ggdb -Wall"
                         debug_static="yes"
